apps/foundation-demo/src/app/design/molecules/sidebar/sidebar-viewport/sidebar-viewport.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="daff-sidebar-viewport {{ mode }}">
+    <div class="daff-sidebar-viewport__sidebar" [@transformSidebar]="_animationState">
+        <ng-content select="daff-sidebar"></ng-content>
+    </div>
+
+    <div class="daff-sidebar-backdrop" 
+        *ngIf="hasBackdrop()"
+        [@fadeBackdrop]="_animationState"
+        [class.daff-sidebar-backdrop--transparent]="!backdropIsVisible" 
+        (click)="_onBackdropClicked()" ></div>
+
+    <div class="daff-sidebar-viewport__content" [@transformContent]="_animationState">
+        <ng-content select="daff-sidebar-content"></ng-content>
+        <daff-sidebar-content *ngIf="!_content">
+            <ng-content></ng-content>
+        </daff-sidebar-content>
```

Comment:
I'm not really sure what lines 13-16 are for. I'm guessing it has to do with a more complex use case? Right now, a plain `<ng-content></ng-content>` does the same thing as these four lines.

Reply:
The `daff-sidebar-content` does nothing currently -- it was an artifact of me trying to understand @angular/material2 structured their sidenav/drawer. I'm going to remove `13` there's no need for that line since I don't expect most people will use `daff-sidebar-content on their own -- they'll assume that you just do. 

The use case on the @angular/material side, as a I understand it, is to change the background color on the `mat-sidenav-content` on top of the stacking context that they use in their implementation. Since we don't use it i'm just going to remove it.

Reply:
Removed in: 2a0fa483b9b4aeefbccd1f693e7063c568d7e64f